src/javascript/ast-value.ts

```diff
@@ -196,4 +197,23 @@ export function getIdentifierName(node: estree.Node): string|undefined {
   }
 }
 
+/**
+ * Formats the given identifier name under a namespace, if one is mentioned in
+ * the commentedNode's comment. Otherwise, name is returned.
+ */
+export function getNamespacedIdentifier(
+    name: string, docs?: jsdoc.Annotation): string {
+  if (!docs) {
+    return name;
+  }
+  const namespace = jsdoc.getTag(docs, 'memberof', 'description');
+  if (namespace) {
+    const rightMostIdentifierName = name.substring(name.lastIndexOf('.') + 1);
```

Comment:
hmm.. ultimately I think we might want to call `getIdentifierName` (or similar) in this function, so that given jsdocs and a node to extract a name from, this contains all the logic in one place to determine the fully qualified name - rather than building up a dot-separated name in `getIdentifierName`, just to split it apart here.

Don't block though... just add a TODO